src/element.ts

```diff
@@ -210,7 +209,12 @@ export class AtomTerminal extends HTMLElement {
     }
     this.terminal.open(this)
     if (atom.config.get("terminal.webgl")) {
+      const { WebglAddon } = await import("xterm-addon-webgl")
       this.terminal.loadAddon(new WebglAddon())
+    } else {
+      // ligatures only work if WebGL is disabled
+      const { LigaturesAddon } = await import("xterm-addon-ligatures")
+      this.terminal.loadAddon(new LigaturesAddon())
     }
```

Comment:
Ligatures aren't currently supported in the webgl plugin but they might be in the future. Right now enabling them with webgl doesn't hurt anything. I think we should enable them even if webgl is enabled so in the future when webgl does support them we don't have to change any code.

I created an issue to track support https://github.com/xtermjs/xterm.js/issues/3094

Reply:
Should I add a config for ligatures?

Reply:
I would just always enable it since it is just a no-op when webgl is enabled and eventually webgl will support it.

Reply:
I added the config just for consistency with other addons. It defaults to true.

https://github.com/atom-ide-community/terminal/pull/14/commits/6a5c437661f0e45e2cbfbb36858af20cdf41a52f